pyvista/core/utilities/points.py

```diff
@@ -182,14 +182,17 @@ def lines_from_points(points, close=False):
 def fit_plane_to_points(points, return_meta=False):
     """Fit a plane to a set of points using the SVD algorithm.
 
+    The plane is automatically sized and oriented to fit the extents of
+    the points.
+
```

Comment:
@user27182
I believe that this change is a bug fix and that the previous behavior is inappropriate, is that correct? If you like, we will fix the pull request label to bug.

Reply:
Yes I would agree it's more of a bug fix than a feature. Please go ahead and fix the label. Let me know if you need me to make any changes directly myself.